src/components/Navigation.vue

```diff
@@ -223,5 +228,23 @@ to {
 :deep(.settings-button) {
 	opacity: .7 !important;
 	font-weight: bold !important;
+	z-index: 1;
+}
+.outbox-border {
+	border-top: 1px solid var(--color-background-darker);
+}
+.outbox {
+	margin-left: 8px;
+	height: 54px;
```

Comment:
why does this need a fixed height? can't it size automatically?

Reply:
you are right, most probably not, i needed the height because i wanted to style it with the fade in effect, but it doest work for open state of settings so i move on with border for now.